src/EnergyPlus/test_ep_as_library.cc

```diff
@@ -63,16 +62,17 @@ void progress_callback_handler(int const progress)
 int main(int argc, char *argv[])
 {
     std::cout << "Using EnergyPlus as a library." << std::endl;
-    EnergyPlus::EnergyPlusData state;
-    StoreMessageCallback(state, message_callback_handler);
-    StoreProgressCallback(state, progress_callback_handler);
+    EnergyPlus::EnergyPlusData *state{reinterpret_cast<EnergyPlus::EnergyPlusData *>(stateNew())};
```

Comment:
I started changing this and then realized that this file must have created an EnergyPlusData object rather than an EnergyPlusState because that's the type `StoreMessageCallback`, `StoreProgressCallback`, and `RunEnergyPlus` have to take as input arguments. So by using the DLL's creation mechanism (`stateNew`) we're stuck with a reinterpret_cast somewhere.